nplbam/app/__init__.py

```diff
@@ -13,11 +13,13 @@ def create_app():
     app.testing = True
     # Set the secret key from the config file
     app.secret_key = config.SECRET_KEY
-    from . import main, animals
+    from . import main, animals, newAnimal
```

Comment:
Is there a reason why we are using lower-camel-case here? Currently the rest of the project is using snake case, and I believe in the style of python snake case may make more sense in this instance.
https://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python